test/functional/mempool_persist.py

```diff
@@ -72,20 +66,24 @@ def run_test(self):
         self.nodes = []
         self.nodes.append(start_node(0, self.options.tmpdir))
         self.nodes.append(start_node(1, self.options.tmpdir))
-        assert_equal(len(self.nodes[0].getrawmempool()), 5)
-        assert_equal(len(self.nodes[1].getrawmempool()), 0)
+        # Give bitcoind a second to reload the mempool
+        time.sleep(1)
+        assert wait_until(lambda: len(self.nodes[0].getrawmempool()) == 5)
+        assert wait_until(lambda: len(self.nodes[1].getrawmempool()) == 0)
```

Comment:
No - this could just as well be a straight assert that node1's mempool is empty. The `time.sleep(1)` above is what actually guarantees that we're not just hitting a window before node1's mempool fills up.

I changed this to a `wait_until()` call for consistency with the line above, but could quite easily change it back if you think that would be clearer for the reader.